src/main/java/com/github/fabienrenaud/jjb/stream/Serialization.java

```diff
@@ -165,4 +166,17 @@ public Object purejson() throws Exception {
         Value purejson = JSON_SOURCE().streamSerializer().purejson(JSON_SOURCE().nextPojo());
         return Generator.create().generate(purejson);
     }
+
+    @Benchmark
+    @Override
+    public Object antons() throws Exception {
+        sk.antons.json.JsonValue jso = JSON_SOURCE().streamSerializer().antons(JSON_SOURCE().nextPojo());
+
+        ByteArrayOutputStream baos = JsonUtils.byteArrayOutputStream();
+        Writer w = new BufferedWriter(new OutputStreamWriter(baos));
```

Comment:
Remove `BufferedWriter`. 
It's useful to minimize I/O when writing to disk, but here the actual output stream writes a byte array which already has the needed capacity. Using `BufferedWriter` here will only contribute to degraded benchmark performance as it introduces an unnecessary intermediate buffer before copying data to the final byte array.

Reply:
done